src/clm/commands/add_carbon.py

```diff
@@ -32,16 +32,25 @@ def add_carbon(input_file, output_file, seed=None):
     # open buffer
     f = open(output_file, "a+")
     # write header
-    row = "\t".join(["input_smiles", "mutated_smiles", "mass", "formula"])
+    row = "\t".join(["input_smiles", "mutated_smiles", "mass", "formula", "inchikey"])
     _ = f.write(row + "\n")
     f.flush()
 
     # read the input SMILES
-    smiles = read_file(input_file, smile_only=True)
-
-    # calculate inchikeys
-    train_mols = [clean_mol(smile, raise_error=False) for smile in smiles]
-    train_inchi = [Chem.inchi.MolToInchiKey(mol) for mol in train_mols if mol]
+    smiles = read_file(input_file, smile_only=False)
```

Comment:
somewhat awkward way to detect if we're dealing with a csv file or a file of smiles. Will be removed once `read_smiles` can handle both cases (by utilizing `pd.read_csv` internally).